src/renderer/corporate/common/corporate-org-chart.jsx

```diff
@@ -91,7 +96,7 @@ const CorporateOrgChart = withStyles(styles)(props => {
 					<hr className={classes.hr} />
 					<CardContent className={classes.orgChart}>
 						<OrgChart
-							tree={generateStructFromCap(profile.entityName, cap)}
+							tree={generateStructFromCap(profile, shareholders)}
```

Comment:
We agreed to remove org chart because it is more relevant to corporate structure inside the company. Like who manages who.

Reply:
Ok, component is kept but commented out of overview page: bb4bb92